modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+
+<portlet-app version="2.0" xmlns="http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd http://java.sun.com/xml/ns/portlet/portlet-app_2_0.xsd">
+	<portlet>
+		<portlet-name>1</portlet-name>
+		<display-name>sample-bean-portlet</display-name>
+		<portlet-class>com.liferay.bean.portlet.cdi.extension.sample.portlet.HelloWorldFacesPortlet</portlet-class>
+		<init-param>
+			<name>javax.portlet.faces.defaultViewId.view</name>
+			<value>/WEB-INF/views/view.xhtml</value>
+		</init-param>
+		<expiration-cache>0</expiration-cache>
+		<supports>
+			<mime-type>text/html</mime-type>
+		</supports>
+		<portlet-info>
+			<title>sample-bean-portlet</title>
+			<short-title>sample-bean-portlet</short-title>
+			<keywords>sample-bean-portlet</keywords>
+		</portlet-info>
+		<security-role-ref>
```

Comment:
@Kvale1733, please rename sample-bean-portlet for Sample Bean Portlet. Refer to https://github.com/liferay/liferay-plugins/blob/7.0.x/portlets/google-adsense-portlet/docroot/WEB-INF/portlet.xml and our portlets with defined as osgi service(for example, JournalContentPortlet). So please modify it in the `<display-name>` and `<portlet-info>`